src/meta/src/barrier/mod.rs

```diff
@@ -1032,6 +1068,40 @@ struct BarrierCompleteOutput {
 }
 
 impl CheckpointControl {
+    fn apply_command_to_tracker(&mut self, epoch_node: &EpochNode) -> (bool, Vec<TrackingJob>) {
```

Comment:
Can you please add doc to the function showing the semantics of the return value?